chapter03/c03_study.rpy

```diff
@@ -96,81 +93,81 @@ label c03_study:
+    With the limited light, I squinted in the dark, making out figures when my eyes met a pair of dangerous green ones.
+    boy "..."
     c "Ah!"
     The boy's tense and dangerous eyes caught me off guard, causing myself to straighten up.
     c "I-I'm sorry! [Pause] I-I let myself in after knocking on your door! I should have waited for you to say something! I-I'm sorry for intruding!"
-    Boy "...?"
+    boy "...?"
     The boy began to relax, his once dangerous eyes slowly turning into curious ones.
     c "Ah... Um..."
     c "My name is Celia... [Pause] Celia Creed."
-    Boy "...!"
+    boy "...!"
     I blinked a few times to help me adjust to the darkness, allowing me to see the outline of the boy's dark hair.
     His hair was so dark, I couldn't even see it earlier. [Pause] It looked as if the darkness was eating a part of his face.
     c "Um, I-I just wanted to welcome you to the hospital!"
     c "I heard that you were in an accident before you arrived here, and that's how you lost your arm... [Pause] I'm sorry that it happened to you."
     c "But, don't worry! [Pause] Everyone here will take good care of you and everything will be okay!"
-    Boy "..."
-    The boy snorted softly as his eyes softened, turning his head towards the curtained window next to him.
+    boy "..."
+    The boy snorted softly as his eyes softened, and he turned his head towards the curtained window next to him.
     I fell silent at his actions, looking down at my feet as I thought about what to say.
-    To him, I'm just a stranger who had just rudely entered his room without permission.
+    To him, I'm just a stranger who had rudely entered his room without permission.
     That won't make him feel at home here...
     c "Um... [Pause] Oh!"
     c "I-If you ever need someone to talk to, feel free to visit room 302! [Pause] That's my room!"
     The boy did nothing as he continued to stare at the curtains.
-    I fell silent by this, curious as to why he didn't open them.
+    I fell silent at this, curious as to why he didn't open them.
     The room is so dark, it's really hard to see. [Pause] So why doesn't he open the curtains?
     I pressed my lips together as I began to glance around the room, feeling immobile for some reason.
     c "...Hm?"
     My wondering eyes stopped as I noticed a tray of food sitting on the ground, in front of the bed.
-    I walked towards the tray, my eyes locked on it- trying to make out every single detail through the darkness.
+    I walked towards the tray, my eyes locked on it - trying to make out every single detail through the darkness.
     c "Hey... [Pause] Did you not eat your lunch? It's sitting here, in front of your bed."
     I knelt down in front of the food to take a better look.
-    The food on the plate consisted of a small portion of rice that had grown cold when you inch your hand near it, a bowl of vegetables and meat, and a cup of warm water that sat in front of three small pills.
```

Comment:
Wasn't quite sure how to work 'when you inch your hand near it' into this sentence so I dropped it